modules/core/src/main/scala/org/tessellation/Main.scala

```diff
@@ -41,6 +46,13 @@ object Main
   val kryoRegistrar: Map[Class[_], KryoRegistrationId[KryoRegistrationIdRange]] =
     coreKryoRegistrar
 
+  override def getMajorityPeerIds(
```

Comment:
afaik we will never use this on the Global L0, so imo it shouldn't be there. it can be an object function imported in other layers, not necessary declared in the tessellation io app that is shared across all layers

Reply:
`getMajorityPeerIds` moved to object